lib/runtime-core/src/sys/windows/memory.rs

```diff
@@ -33,7 +33,7 @@ impl Memory {
 
         let protect = protection.to_protect_const();
 
-        let ptr = unsafe { VirtualAlloc(ptr::null_mut(), size, MEM_RESERVE | MEM_COMMIT, protect) };
```

Comment:
This still works, even though we're not committing the memory?

Reply:
yes